test/python/circuit/test_circuit_operations.py

```diff
@@ -478,6 +478,33 @@ def test_copy_copies_registers(self):
         self.assertEqual(len(qc.cregs), 1)
         self.assertEqual(len(copied.cregs), 2)
 
+    def test_copy_empty_like_circuit(self):
+        """Test copy_empty_like method makes a clear copy."""
```

Comment:
Could you also check metadata and calibrations, since keeping these were among the motivation for this method? 🙂